website/app/components/doc/cards/card.hbs

```diff
@@ -1,7 +1,14 @@
 <li class={{this.classNames}} ...attributes>
   {{#if @route}}
     <LinkTo class="doc-cards-card__link" @route={{@route}} @model={{@model}}>
-      <img class="doc-cards-card__image" src={{@image}} alt="" role="presentation" />
+      <img
+        class="doc-cards-card__image"
+        src={{@image}}
+        alt={{(concat @title " - " @caption)}}
```

Comment:
@MelSumner this triggered a linting error so it seems like I should undo this. Can you confirm?

Reply:
@Dhaulagiri what was the linting error? Might be a false positive, so I'd just add an inline ignore if so. But the way that we use most of the card images, they are superfluous anyway so I'd probably do an empty alt and a role of none.

Reply:
i recall that we discussed this at the time and i removed the alt tag entirely